04-onion-routing.md

```diff
@@ -501,9 +504,15 @@ The construction returns a single 1366-byte packet along with the first receivin
 The packet construction is performed in the reverse order of the route, i.e.
 the last hop's operations are applied first.
 
-The packet is initialized with 1366 `0x00`-bytes.
+The packet is initialized with 1366 _random_ bytes derived from a CSPRNG
```

Comment:
```suggestion
The packet is initialized with 1300 _random_ bytes derived from a CSPRNG
```
The actual encrypted packet is only 1300 bytes, the remainder are version, ephemeral key and HMAC. No need to overgenerate :-)